### PR TITLE
Add reinvest rewards script

### PR DESCRIPTION
**Pull request by kaymakf**

### CODE REVIEW

File: projects/reinvest-rewards/index.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Reinvest Rewards Script",
+    "type": "Tools",
+    "image": "deus.png",
+    "state": 1,
+    "twitter_link": "twitter.com/deuslabs",
+    "tg_link": "",
```
Comment (orkunkl) [inline projects/reinvest-rewards/index.json]: deus tg link

Comment (kaymakf) [reply projects/reinvest-rewards/index.json]:
```suggestion
    "tg_link": "https://t.me/deuslabs",
```

File: projects/reinvest-rewards/index.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "address": "",
+    "github_link": "https://github.com/deus-labs/juno-scripts",
+    "website_link": ""
```
Comment (kaymakf) [inline projects/reinvest-rewards/index.json]:
```suggestion
    "website_link": "https://deuslabs.fi/"
```

File: projects/reinvest-rewards/index.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Reinvest Rewards Script",
+    "type": "Tools",
+    "image": "deus.png",
+    "state": 1,
+    "twitter_link": "twitter.com/deuslabs",
```
Comment (kaymakf) [inline projects/reinvest-rewards/index.json]:
```suggestion
    "twitter_link": "https://twitter.com/deuslabs",
```

**Review by orkunkl — APPROVED** [review]

**Review by blockcreators — APPROVED** [review]

**Pull request merged**